website/data/docs-navigation.js

```diff
@@ -144,6 +144,10 @@ export default [
             category: 'conversion',
             content: ['can', 'convert', 'try'],
           },
+          {
+            category: 'vault',
+            content: ['vault'],
+          },
```

Comment:
Nice, do you think it could be a good idea to have a hashitools folder though ? 

Something like : 
* `functions/hashicorp/vault`
* `functions/hashicorp/consul`